java/vector/src/main/java/org/apache/arrow/vector/TypeLayout.java

```diff
@@ -217,11 +220,16 @@ public TypeLayout(BufferLayout... bufferLayouts) {
     this(asList(bufferLayouts));
   }
 
-
+  /**
+   * Returns the individual {@linkplain BufferLayout}s for the given type.
+   */
   public List<BufferLayout> getBufferLayouts() {
     return bufferLayouts;
   }
 
+  /**
+   * Returns the types of each buffer for in this layout.
```

Comment:
remove for or in?

Reply:
removed in.  added a clarifying sentence as a follow-up.